cmd/up/main.go

```diff
@@ -254,18 +255,18 @@ func addCustomQueryRunGroup(ctx context.Context, g *run.Group, l log.Logger, opt
 								"warnings", fmt.Sprintf("%#+v", warn),
 								"err", err,
 							)
-							m.CustomQueryErrors.WithLabelValues(queryType, name).Inc()
+							m.CustomQueryErrors.WithLabelValues(queryType, name, strconv.Itoa(httpCode)).Inc()
```

Comment:
I'm still thinking about https://github.com/observatorium/up/pull/35#pullrequestreview-763060440, i.e. we could pass the counter to `query` instead and increment when we get a response from server - that way we wouldn't have to propagate the HTTP code all the way to this top method. What do you think @spaparaju ?